calculator.rb

```diff
@@ -0,0 +1,106 @@
+# acceptable operators
+#this is inelegant I know, I ran out of time!
+operators = [ "\+","\-","\*","\/","divide","DIVIDE","Divide","Div","DIV",
+  "div","add","addition","Add","ADD","ADDITION","Addition",
+  "subtract","Subtract","sub","Sub","SUB","SUBTRACT","mult",
+  "multiply","Multiply","MULTIPLY"
```

Comment:
I like the thoroughness here. One thing you could do to make it a little simpler is to always convert the operation to `upcase` or `downcase` before checking it against this list.